### DISCUSSION TITLE
Facebook, Twitter,Whatsapp and LinkedIn icons not there

### DISCUSSION THREAD

**Discussion by Saswat689** [tailwindlabs/heroicons#1024] [Ideas]
I can't seem to find social media icons by searching on the heroicons site. This is strange provided the popularity and usability of these icons. Am I missing something or does heroicons does not provide these yet ?

**Comment by zefir-git** [comment]
Why burden Heroicons with brands/logos when https://simpleicons.org/ is specifically for this purpose
